src/platform/Zephyr/BLEAdvertisingArbiter.h

```diff
@@ -61,6 +61,17 @@ struct Request : public sys_snode_t
     OnAdvertisingStopped onStopped;       ///< (Optional) Callback invoked when the request stops being top-priority.
 };
 
+/**
+ * @brief Initialize BLE advertising arbiter
+ *
+ * @note This method must be called before trying to insert or cancel any requests.
+ *
+ * @param btId   Local Bluetooth LE identifier to be used for the advertising parameters.
```

Comment:
is there any `Uninit/Shutdown` that will set "initialized" to false?

or if not, it seems that the btId is sticky and forever.  Maybe this should be explained in the documentation and why this lack of shutdown is correct.

Reply:
Well, similarly to many other components (like BLEManager) it was not designed to be uninitialized and the assumption is it will be run once and work until reboot. 

> it seems that the btId is sticky and forever
Yes, indeed I also spotted it and I'm going to fix this in the next PR (I can create issue for improvement to track this). I believe that Bluetooth ID could be a part of `Request` structure that is passed to the `InsertRequest` method, so different bluetooth ids could be used for the following requests.

Reply:
I updated docs with the information that for now id is fixed and created issue to improve it: https://github.com/project-chip/connectedhomeip/issues/33310